react/features/base/profile/middleware.js

```diff
@@ -31,13 +34,11 @@ MiddlewareRegistry.register(store => next => action => {
  * @returns {void}
  */
 function _updateLocalParticipant(store) {
+    const localParticipant = getLocalParticipant(toState(store));
     const profile = getProfile(toState(store));
```

Comment:
Please note that `toState(store)` is executed twice in the two lines above! That's very likely the result of copy-pasting that that's why copy-pasting is discouraged by our coding style.

I'm taking care of this one.